src/middlewares/createInsightsMiddleware.ts

```diff
@@ -6,6 +7,7 @@ import type {
 } from '../types';
 import { getInsightsAnonymousUserTokenInternal } from '../helpers';
 import { warning, noop, getAppIdAndApiKey, find } from '../lib/utils';
+import connectConfigure from '../connectors/configure/connectConfigure';
```

Comment:
what about the configure widget instead of connector? It doesn't have any warnings or validation that would make the impact bigger

Reply:
Not sure which warnings or validation you mean. My initial thought on using the connector was to avoid any widget within middlewares. The connctors, middlewares, utils, etc can be shared across flavors, but we won't share widgets (although configure widget is renderless.)